SpatialGDK/Documentation/content/helper-scripts.md

```diff
@@ -1,8 +1,7 @@
 <%(TOC)%>
 # Helper scripts
 
-These scripts are located under `Plugins\UnrealGDK\SpatialGDK\Build\Scripts\` directory of your game.
-
-| Helper script | Parameters | Description |
-| --- | --- | --- |
-| `BuildWorker.bat` | `<target> <platform> <configuration> <YourGame>.uproject [--skip-codegen]` | Example: </br> `BuildWorker.bat ExampleGameEditor Win64 Development ExampleGame.uproject` </br></br> Build, cook and zip your Unreal server-workers and client-workers for use with a SpatialOS cloud deployment (uploaded using [`spatial cloud upload`](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud)). </br></br>  The following `<target>`s  generate zipped workers: </br> - `<YourGame>` </br> - `<YourGame>Server` </br> - `<YourGame>Editor` </br></br> Any other `<target>` passes all arguments to `Engine\Build\BatchFiles\Build.bat` with no cooking or zipping performed.|
+| Helper script | Location (relative to `UnrealGDK` folder) | Parameters | Description |
+| --- | --- | --- | --- |
+| `BuildWorker.bat` | `SpatialGDK\Build\Scripts\` | `<target> <platform> <configuration> <YourGame>.uproject [--skip-codegen]` | Example: </br> `BuildWorker.bat ExampleGameEditor Win64 Development ExampleGame.uproject` </br></br> Build, cook and zip your Unreal server-workers and client-workers for use with a SpatialOS cloud deployment (uploaded using [`spatial cloud upload`](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud)). </br></br>  The following `<target>`s  generate zipped workers: </br> - `<YourGame>` </br> - `<YourGame>Server` </br> - `<YourGame>Editor` </br></br> Any other `<target>` passes all arguments to `Engine\Build\BatchFiles\Build.bat` with no cooking or zipping performed.|
```

Comment:
I know this isn't changed but it's quite cryptic so in the spirit of good packaging, we should make it clearer, I think! :-)

"The following <target>s generate zipped workers:" <- This isn't clear - I think we mean that depending on the target of the script, you can build server workers (<YourGame>Server ) or client workers (<YourGame>Editor) or  both (<YourGame>). 

Is the <platform> the destination platform of built client workers only?  
What are the platform options? Is it only Win64?
Do we need a platform for server-workers?

What is the <configuration> parameter do and are there any other options for the configuration?

What do we mean by cooking? How is this different from building? 

What does the parameter [--skip-codegen] do? Why do I want to use it or not use it?
Are there any consequences to zipping? or not zipping? Does it affect the upload?

"uploaded using..." - I am guessing that the script doesn't do the uploading, but it's something the user still does via spatial CLI commands?

Reply:
Have put all these comments in a GDoc, against the rendered MD, because it's easier to discuss there: https://docs.google.com/document/d/1DLrLcDsqrxhLpw1L_GNBGwoQR6kRjpWKr4-reUwsRYU/edit